src/components/message/demo/message-with-close.twig

```diff
@@ -0,0 +1,7 @@
+{% embed '../message.twig' with { includeClose: true } %}
```

Comment:
Nit: I am _woefully_ bad at remembering this and there are merged examples of the contrary, but Twig files should generally favor [underscores over camelcase](https://twig.symfony.com/doc/3.x/coding_standards.html).

(I've got a to-do to create an issue for the existing violations to this, but I keep thinking I need to write this info down first, and then something else comes up... 😅)